apps/etterlatte-behandling/src/main/kotlin/inntektsjustering/AarligInntektsjusteringJobbService.kt

```diff
@@ -155,6 +167,31 @@ class AarligInntektsjusteringJobbService(
         return null
     }
 
+    private suspend fun opprettRevurderingOgOppgave(
+        sakId: SakId,
+        loependeFom: YearMonth,
+        forrigeBehandling: Behandling,
+    ) {
+        val persongalleri = grunnlagService.hentPersongalleri(forrigeBehandling.id)
+        revurderingService
+            .opprettRevurdering(
+                sakId = sakId,
+                persongalleri = persongalleri,
+                forrigeBehandling = forrigeBehandling.id,
+                mottattDato = null,
+                prosessType = Prosesstype.MANUELL,
+                kilde = Vedtaksloesning.GJENNY,
+                revurderingAarsak = Revurderingaarsak.INNTEKTSENDRING, // TODO ny årsak
+                virkningstidspunkt = loependeFom.atDay(1).tilVirkningstidspunkt("Årlig inntektsjustering"),
+                utlandstilknytning = forrigeBehandling.utlandstilknytning,
+                boddEllerArbeidetUtlandet = forrigeBehandling.boddEllerArbeidetUtlandet,
+                begrunnelse = "Årlig inntektsjustering",
```

Comment:
pirk, men begrunnelse kan vel defineres en plass da samme tekst brukes to plasser

Reply:
Jepp, jeg tenker også å høre med Mari og Liv Inger hva de tenker teksten burde være